lib/src/execute.rs

```diff
@@ -511,8 +511,10 @@ impl ExecuteCtx {
                 // due to wasmtime limitations, in particular the fact that `Instance` is not `Send`.
                 // However, the fact that the module itself is created within `ExecuteCtx::new`
                 // means that the heavy lifting happens only once.
-                let mut store = create_store(&self, session, profiler, |_| {})
-                    .map_err(ExecutionError::Context)?;
+                let mut store = create_store(&self, session, profiler, |ctx| {
+                    ctx.arg("compute-app");
```

Comment:
Do you also want to add this for the `ComponentCtx::create_store` case as well?

Reply:
Good catch!

Reply:
I've added this for components and enabled testing for it, but the adapter will need some changes to get the test passing. I'll put that on my TODO list for component parity, but don't think the adapter changes should block this PR.